docs/deploy-rbd.md

```diff
@@ -55,7 +55,8 @@ make image-cephcsi
 | `dataPool`                                                                                          | no                   | Ceph pool used for the data of the RBD images.                                                                                                                                                                          |
 | `volumeNamePrefix`                                                                                  | no                   | Prefix to use for naming RBD images (defaults to `csi-vol-`).                                                                                                                                                           |
 | `snapshotNamePrefix`                                                                                | no                   | Prefix to use for naming RBD snapshot images (defaults to `csi-snap-`).                                                                                                                                                 |
-| `imageFeatures`                                                                                     | no                   | RBD image features. CSI RBD currently supports only `layering` feature. See [man pages](http://docs.ceph.com/docs/nautilus/man/8/rbd/#cmdoption-rbd-image-feature)                                                         |
+| `imageFeatures`                                                                                     | no                   | RBD image features. CSI RBD currently supports only `layering` feature. See [man pages](http://docs.ceph.com/docs/nautilus/man/8/rbd/#cmdoption-rbd-image-feature)                                                      |
+| `mapOptions`                                                                                        | no                   | Map options to use when mapping rbd image. See [man pages](https://docs.ceph.com/docs/octopus/man/8/rbd/#kernel-rbd-krbd-options).                                                                                      |
```

Comment:
Nit: link to the master version (i.e. https://docs.ceph.com/docs/master/man/8/rbd/#kernel-rbd-krbd-options), here and on the previous line.